code/modules/necromorph/corruption/cyst.dm

```diff
@@ -17,12 +17,12 @@
 	plane = EFFECTS_BELOW_LIGHTING_PLANE //Cysts need to draw over many things
 	var/obj/item/ammo_casing/cystbomb/payload = null
 
-	biomass = 5
+	biomass = 15
 	biomass_reclamation = 1
-	reclamation_time = 5 MINUTES
+	reclamation_time = 7.5 MINUTES
 
```

Comment:
These aren't minor changes, maybe in the right direction but excessive. Do this instead:
Biomass cost: 8
reclamation change is fine
recharge delay, leave it as is, 5 seconds is precisely based on the games, that won't ever change

Reply:
5 seconds is too short and too spammy. Especially done if spammed out the ass.

Reply:
Cysts have a lot of weaknesses. 
-Any cyst caught in their blast (including themselves) is destroyed. 
-You can trigger their firing by tossing something infront of them. Chuck a lighter or an empty mag to create safe passage
-Their limited arc of fire makes them easy to flank
-Their bombs can be picked up with kinesis and throw back, or at necros.
-They have a range limit of 6, meaning anyone with any gun can stand just at the edge of their range and snipe them

they're an area denial tool but far from invulnerable, they can kill quickly if someone walks into a trap, thats working as intended. Cysts have recently been buffed significantly because they were a joke before, their current state isn't far off right

Reply:
You do realize 5 seconds is nothing right? You can't walk 2 feet without it firing again.

Also, you want authenticity? Power to you. But authenticity vs gameplay effects have to be put into consideration, and you're not doing a whole lot by trying to defend their current state. You turned them from a joke to Tripod-worthy.

Reply:
let us worry about design. please amend the cyst changes as described above

Reply:
I've changed it to 6.5. I implore you to try this out at least. Cysts are now spammed out the ass, and adjustments have to be made.

Reply:
adjustments are needed, but not like this. 
Let me handle it, you don't have to worry about it. Just remove all cyst related changes, i'll make a PR today